RELEASE.md

```diff
@@ -193,3 +195,36 @@ To merge a release branch `release-X.Y` into `main`, please do the following:
 - Once approved, merge the PR with a **Merge** commit through one of the following strategies:
   - Temporarily enable "Allow merge commits" option in "Settings > Options"
   - Locally merge the `merge-release-X.Y-to-main` branch into `main`, and push the changes to `main` back to GitHub. This doesn't break `main` branch protection, since the PR has been approved already, and it also doesn't require removing the protection.
+
+## Helm release
+
+### Pre-requisites:
+
+Either:
+
+- Both Mimir and Enterprise Metrics (GEM) have container images released to Docker Hub that line up (same weekly, same RC, same release version).
+- Mimir and GEM images do not match, but there are no breaking changes between the two. That is all shared configuration parameters behave the same way. For example GEM has a bugfix in a maintenance version.
+
+### Release process
+
+1. Determine version number
+
+   - Helm [mandates semantic versioning](https://helm.sh/docs/topics/charts/#the-chartyaml-file)
+   - Weekly release should have the version: x.y.z-weekly.w (e.g. 3.1.0-weekly.196)
+   - Release candidate: x.y.z-rc.w (e.g. 3.1.0.rc.1)
+   - Final version: x.y.z (e.g. 3.1.0)
+
+   **Note**: the weekly and RC number must be separated with dot (.) for correct version ordering.
+
+1. To make a release you have to merge a version bump to the Helm chart to a branch that allows Helm release (e.g. main, release-x.y), see current list in [helm-release.yaml](https://github.com/grafana/mimir/blob/main/.github/workflows/helm-release.yaml) github action.
+   - The release process checks and creates a git tag formatted as (mimir-distributed-_version_). The process fails if the tag already exists
+   - The process creates the release in the [Grafana helm-charts](https://grafana.github.io/helm-charts/) Helm repository
+   1. Open a PR
```

Comment:
This has a mix of ordered and unordered lists. Did you mean to make the ordered list a sublist of the unordered one?

Reply:
I meant to give some context to what will happen, rewording now.

Reply:
wdyt now?